test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -447,18 +455,21 @@ def forward(self, x):
         mod = M().eval()
         v = torch.randn((1, 3, 8, 8), dtype=torch.float32, requires_grad=False).add(1)
 
-        # Totally pattern_matcher_count 2,
-        # pattern_matcher_nodes 8
-        # 1. pair of to_int8 and to_fp32 at conv input matched in pointless_convert pass
-        #    at torch/_inductor/fx_passes/joint_graph.py: [convert_element_type, convert_element_type_1]
-        # 2. dequant-conv pattern matched in quantization weight prepack
-        #    [convert_element_type_1, sub, mul_1, dequantize_per_channel, clone, convolution]
+        def matcher_check_fn():
+            # 1. Dequant-Conv2D pattern matched in QConv2D weight prepack * 1
+            #    [convert_element_type_1, sub, mul_1, dequantize_per_channel, clone, convolution]
```

Comment:
how do we get this list?

Reply:
By printing the `m.nodes` here, we can the nodes list in each matched pattern https://github.com/pytorch/pytorch/blob/042445b7d3c7e7d4c1c66bcb3e5d35498bd2a4fb/torch/_inductor/pattern_matcher.py#L1137